core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -152,123 +152,117 @@ var TopicEditorPage = function() {
 
   this.expectNumberOfQuestionsForSkillWithDescriptionToBe = async function(
       count, skillDescription) {
-    await waitFor.elementToBeClickable(
-      selectSkillDropdown, 'Skill select dropdown takes too long to appear.');
-    await selectSkillDropdown.click();
-    await element(by.css('option[label="' + skillDescription + '"]')).click();
+    await action.click('Select skill dropdown', selectSkillDropdown);
+    var skillDescriptionButton = element(
+      by.css('option[label="' + skillDescription + '"]'));
+    await action.click('Skill description button', skillDescriptionButton);
     await waitFor.visibilityOf(
       questionItems.first(), 'Question takes too long to appear');
     expect(await questionItems.count()).toEqual(count);
   };
 
   this.saveQuestion = async function() {
     await general.scrollToTop();
-    await saveQuestionButton.click();
-    await waitFor.invisibilityOf(
```

Comment:
Why remove this?

Reply:
Removed by oversight. Mistook the waitFor.invisibilityOf(..) for waitFor.visibilityOf(..). Reverted back to original.